cmake/modules/FindUMFPACK.cmake

```diff
@@ -109,14 +105,22 @@ ENDIF()
 
 #
 # Link against everything we can find to avoid underlinkage:
+# (also use lib- prefixes for Windows)
 #
 FIND_UMFPACK_LIBRARY(UMFPACK umfpack)
+FIND_UMFPACK_LIBRARY(UMFPACK libumfpack)
 FIND_UMFPACK_LIBRARY(AMD amd)
+FIND_UMFPACK_LIBRARY(AMD libamd)
 FIND_UMFPACK_LIBRARY(CHOLMOD cholmod)
+FIND_UMFPACK_LIBRARY(CHOLMOD libcholmod)
 FIND_UMFPACK_LIBRARY(COLAMD colamd)
+FIND_UMFPACK_LIBRARY(COLAMD libcolamd)
 FIND_UMFPACK_LIBRARY(CCOLAMD ccolamd)
+FIND_UMFPACK_LIBRARY(CCOLAMD libccolamd)
 FIND_UMFPACK_LIBRARY(CAMD camd)
+FIND_UMFPACK_LIBRARY(CAMD libcamd)
 FIND_UMFPACK_LIBRARY(SuiteSparse_config suitesparseconfig)
+FIND_UMFPACK_LIBRARY(SuiteSparse_config libsuitesparseconfig)
```

Comment:
I see - I bet I do something wrong then - if these lines are not there, cmake does not find my libumfpack.lib, if the lines are added, they are found. Could you please verify?